### PR TITLE
Automated chart bump jaeger-operator-2.18.4

### PR DESCRIPTION
**Pull request by d2iq-dispatch**
**What type of PR is this?**
Chore

**What this PR does/ why we need it**:
Bump jaeger.

**Which issue(s) this PR fixes**:

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
bump jaeger-operator-2.18.4
```

**Checklist**

* [ ] *If a chart is changed, the chart version is correctly incremented.*
* [ ] The commit message explains the changes and why are needed.
* [ ] The code builds and passes lint/style checks locally.
* [ ] The relevant subset of integration tests pass locally.
* [ ] The core changes are covered by tests.
* [ ] The documentation is updated where needed.

### CODE REVIEW

File: addons/jaeger/jaeger.yaml [hunk 1]
```diff
@@ -8,13 +8,13 @@ metadata:
...
   annotations:
-    catalog.kubeaddons.mesosphere.io/addon-revision: "2.18.3-4"
+    catalog.kubeaddons.mesosphere.io/addon-revision: "2.18.3-5"
     appversion.kubeaddons.mesosphere.io/jaeger-operator: "1.21.2"
     appversion.kubeaddons.mesosphere.io/jaeger: "1.21.0"
     stage.kubeaddons.mesosphere.io/jaeger: Experimental
     endpoint.kubeaddons.mesosphere.io/jaeger: "/ops/portal/jaeger"
-    docs.kubeaddons.mesosphere.io/jaeger: "https://www.jaegertracing.io/docs/1.21/"
-    values.chart.helm.kubeaddons.mesosphere.io/jaeger: "https://raw.githubusercontent.com/jaegertracing/helm-charts/aa3b160/charts/jaeger-operator/values.yaml"
+    docs.kubeaddons.mesosphere.io/jaeger: "https://www.jaegertracing.io/docs/1.20/"
```
Comment (vespian) [inline addons/jaeger/jaeger.yaml:16]: How is this field set?

@cgawron1 Do you know who now owns the dispatch instance/script that creates this PRs?

Comment (cgawron1) [reply addons/jaeger/jaeger.yaml:16]: This script is run from a cron via Dispatch.   This is the script that is executed 
https://github.com/mesosphere/dispatch-tasks/blob/master/bump_charts/bump_charts.py

bump_charts is owned by DevProd

Comment (orsenthil) [reply addons/jaeger/jaeger.yaml:16]: I find that bump_charts script very hard to understand and read.

Comment (joejulian) [reply addons/jaeger/jaeger.yaml:16]: looks pretty straight-forward to me. /shrug

We can still throw @cprovencher under the bus for it though. He's not here to defend himself. ;)

Comment (orsenthil) [reply addons/jaeger/jaeger.yaml:16]: @joejulian - I take back. yeah, it is not hard and not even a hard regex. Just was not sure why.

Comment (vespian) [reply addons/jaeger/jaeger.yaml:16]: @cgawron1 I took a quick look at the script and did not find anything. I believe that this should be investigate as the version is wrong.

Who should own it? I can take a deeper look into it, but @drewvanstone would need to approve.

Comment (cgawron1) [reply addons/jaeger/jaeger.yaml:16]: The script grabs the version to bump from the apiversion. In this case this version was overwritten by hand by grabbing it from website. This situation is an edge-case and given the human requirement to manually review these bump it is recommended to leave as is. Thanks @joejulian for the insights.

**Comment by joejulian** [comment]
@GoelDeepak I assume you just want to close this?

**Comment by vespian** [comment]
> I assume you just want to close this?

We just need to manually fix this. It was in my queue, but during the release time it was of low priority.

Fixing it now.

**Review by vespian — APPROVED** [review]

**Comment by vespian** [comment]
@mesosphere/sig-ksphere-catalog Can I have a review?

**Review by joejulian — APPROVED** [review]

**Comment by vespian** [comment]
@joejulian @orsenthil Do you know what is the `Mergable` check? It does not allow me to merge PR, and does not clarify which label is missing either :(

**Comment by joejulian** [comment]
@vespian If you go in to Details and look at Mergeable, there's this.
image
We started a really short discussion about making Mergeable add comments to state the same that can be found there. Would that be a desired behavior to you?

**Comment by joejulian** [comment]
If you were an administrator for a production cluster and you wanted to know whether or not to upgrade your addons to this version, is this a valuable release note for you?

Personally, I would prefer to see what features or bug fixes that bump brings so I know whether it's something that helps me. That's what I look for in release notes, fwiw.

**Comment by joejulian** [comment]
As an example:
https://github.com/mesosphere/kubernetes-base-addons/blob/master/RELEASE_NOTES.md#stable-118-300

I think cert-manager is a little bit too verbose, listing items that are trivial and don't apply to users. Those things that do, like "v1 API" don't have enough detail, like "This is a complete replacement of the v1alpha1 CRDs. This upgrade will change all necessary resources to use the new API. This upgrade cannot be reverted."

Dex is good. If I wasn't even noticing a lack of metrics from dex, this would alert me to it and make me want to upgrade.

Istio is way too verbose. It's effectively a copy-paste of upstream's auto-generated release notes that include a lot of trivial notes that have no user-facing impact.

**Comment by vespian** [comment]
> We started a really short discussion about making Mergeable add comments to state the same that can be found there. Would that be a desired behavior to you?

No, the way it is now is way clearer. I am just not used to the `Details` link having useful data in PR status checks.

**Comment by vespian** [comment]
> If you were an administrator for a production cluster and you wanted to know whether or not to upgrade your addons to this version, is this a valuable release note for you?

Depends on how much resources we have at our disposal.

Ideally yes - as an admin, I would appreciate my vendor to summarise the changes for me in just one place. In our current situation though I am struggling to find time to keep the bumps going and do not let #sig-networking slip into abandonment. Networking stuff it is not a priority ATM.

Thanks for understanding.

**Comment by joejulian** [comment]
@mesosphere-mergebot backport release/3.3

**Pull request merged**